src/mscorlib/src/System/Runtime/InteropServices/NonPortable.cs

```diff
@@ -15,7 +15,7 @@ public static int GetHRForException(Exception e)
 
         public static int AddRef(System.IntPtr pUnk)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.Arg_PlatformNotSupported);
```

Comment:
All the ones in this file should have PlatformNotSupported_ComInterop. Note that this file is only included for Unix builds.